README.md

```diff
@@ -1,260 +1,64 @@
-- `-n` to represent the number of Windows nodes to be created for test run
-- `-b` gives an alternative path to the WMCO binary. This option overridden in OpenShift CI.
-       When building the operator locally, the WMCO binary is created as part of the operator image build process and
-       can be found at `build/_output/bin/windows-machine-config-operator`, this is the default value of this option.
-
-       
-Example command to spin up 2 Windows nodes and retain them after test run:
-```
-hack/run-ci-e2e-test.sh -s -n 2      
-```
-
-## Bundling the Windows Machine Config Operator
-This directory contains resources related to installing the WMCO onto a cluster using OLM.
-
-### Pre-requisites
-[opm](https://github.com/operator-framework/operator-registry/) has been installed on the localhost.
-All previous [pre-requisites](#pre-requisites) must be satisfied as well.
-
-### Generating a new bundle
-This step should be done in the case that changes have been made to any of the yaml files in `deploy/`.
-
-If changes need to be made to the bundle spec, a new bundle can be generated with:
-```shell script
-operator-sdk generate csv --csv-version $NEW_VERSION --operator-name windows-machine-config-operator
-```
-
-You should replace `$NEW_VERSION` with the new semver.
-
-Example: For CSV version 0.0.1, the command should be:
-```shell script
-operator-sdk generate csv --csv-version 0.0.1 --operator-name windows-machine-config-operator
-``` 
-This will update the manifests in directory: `deploy/olm-catalog/windows-machine-config-operator/manifests`
-This directory will be used while [creating the bundle image](#creating-a-bundle-image)
-
-After generating bundle, you need to update metadata as well. 
-```shell script
-operator-sdk bundle create --generate-only --channels alpha --default-channel alpha
-```
-
-### Creating a bundle image
-You can skip this step if you want to run the operator for [developer testing purposes only](#development-workflow)
-
-A bundle image can be created by editing the CSV in `deploy/olm-catalog/windows-machine-config-operator/manifests/`
-and replacing `REPLACE_IMAGE` with the location of the WMCO operator image you wish to deploy.
-See [the build instructions](#build) for more information on building the image.
-
-You can then run the following command in the root of this git repository:
-```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$BUNDLE_TAG -d deploy/olm-catalog/windows-machine-config-operator/manifests \
---channels alpha --default-channel alpha --image-builder podman
-```
-The variables in the command should be changed to match the container image repository you wish to store the bundle in.
-You can also change the channels based on the release status of the operator.
-This command should create a new bundle image. Bundle image and operator image are two different images. 
+* OKD/OCP 4.6 cluster running on Azure or AWS, configured with hybrid OVN Kubernetes networking
```

Comment:
Is 4.6 shipping with OVN kubernetes as default? If not we should link to instructions to create a cluster with it.

Reply:
it is not. I'll add back all the links as Aravindh suggested above